documentation/modules/auxiliary/scanner/http/f5_mgmt_scanner.md

```diff
@@ -0,0 +1,32 @@
+## Vulnerable Application
+
+This module scans for web management interfaces of the following F5 Networks devices:
+
+ * BigIP
+ * BigIQ
+ * Enterprise Manager
+ * ARX
+ * FirePass
+
+## Verification Steps
+
+  1. Install the application/hardware
+  2. Start msfconsole
+  3. Do: ```use scanner/http/f5_mgmt_scanner```
+  4. DO: ```set rhosts```
+  5. Do: ```run```
+  6. You will learn if this is a F5 web management interface
```

Comment:
Considering RHOSTS could potentially be multiple machines, I'm not sure that this statement reflects the fact that multiple results could be returned. Could we update it to be in the plural tense?